### ISSUE TITLE
[Modern] FARP, FOB and Base types + logisitics needs clarification and documentation

### ISSUE THREAD

**Issue by madrabbit711** [afinegan/DynamicDCS_BugReports#24]
Suggest trying to promote the following terminology around the DDCS server.  

A) FARP = New numerous areas with helipads that do have a CC but do not have a comms tower and resupply.  In-line, quite correctly, with Drex's comments that FARPs should stay true to their acronym i.e. only rearm and refueling.

B) FOB = Original "FARPs" that have the features of a FARP but also a comms tower and resupply, resulting in non-player controlled base defences.

C) Capturable Airbase = Runway but no helipads (= helos not dependent on resuply trucks).  Has CC, comms tower, resupply = non-player controlled base defences.

D) Neutral Airbase =  Runway but no helipads (= helos not dependent on resuply trucks).  Has CC, but no comms tower = no base defences unless player slung.  Unsure of the proper definition as 'FOB with a runway' is confusing as no comms tower.

In regards to the above types of areas, the following needs definition:

1. If a CC present and if CC is side specific e.g. "CC already exist message" i.e. requires destruction?
2. Define whether helos at FARPs and/or FOBs start with fuel.  Obviously base rely on inherent DCS mechanism within which 
3. Requirements for capture in regards to contesting units (including radius) and if a ground unit is required to capture.
4. Requirements for refueling e.g. do neutral airbases require a friendly ground unit to allow resupply or is the CC enough.

In short, clearer definitions and the above details in a table form would help with the current confusion immensely.  Keep in-mind a lot of players don't know what is being handled by DDCS (Drex) or DDCS inherently e.g. (4) above.

Need to ensure that in regards to the above requirements, that function is somewhat balanced. e.g. FARP/FOBs that have helos without fuel are dependent on trucks but airbases are not.

**Comment by madrabbit711** [comment]
Relates to #23

**Comment by Quaggles** [comment]
I think the term FARP confuses things as there are "FARP pads" at both FARPs and FOB's so we probably need a new name to define the current DDCS terms for FARPs and capture points. I think the term FOB might stay reserved incase they ever make a return, back in the day they were slingable bases that allowed you to spawn units at the front line.

To answer some of your questions:
> If a CC present and if CC is side specific e.g. "CC already exist message" i.e. requires destruction?

Yes, if the enemy CC still exists you will need to kill it before you can put down your CC Build crate and build your own

> Define whether helos at FARPs and/or FOBs start with fuel. Obviously base rely on inherent DCS mechanism within which

This is being overhauled at the moment, in an effort to reduce unit counts every FARP and Capture Point will only have a single FARP pad per team and all helicopters will spawn on the ground with some fuel. This stops annoying situations where a fighter comes in and strafes the fuel trucks which makes the base unusable as those helicopters can now just spawn in and sling new ones from the CC.

> Requirements for capture in regards to contesting units (including radius) and if a ground unit is required to capture.

> Requirements for refueling e.g. do neutral airbases require a friendly ground unit to allow resupply or is the CC enough.

Base neutralising that blocks the enemy from talking to the ground crew (Rearming, refueling, repairing) is controlled on the DCS side and we have no control of it. It usually happens when there are units from both sides on the airfield, this distance can be pretty close on some airfields. As long as there is 1 friendly unit on an airfield and no enemy ones you should be able to refuel.

Capture points and neutral airfields are captured by spawning CC's at them, as long as the CC is alive you should be able to spawn there. Whether you can rearm depends on the rules above.

Airbases and FARPs are captured by just killing all enemy units and moving one of your own ground units there. You do not need to kill the enemy CC or build your own to capture these.

**Comment by madrabbit711** [comment]
(I appreciate that the above and the following is arguing semantics but it important for team comms, especially given the variety of areas being introduced)

>  think the term FOB might stay reserved incase they ever make a return
I think the use of the terms FOBs for these temporary staging points is even more incorrect than utilising it for the current FARPs considering that in terms of size and complexity in real-life:
1) FOBs = multiple helipads + defences
2) FARPs = helipads only no defences
3) staging areas = single pad only no defences

> Yes, if the enemy CC still exists you will need to kill it before you can put down your CC Build crate and build your own
Then this is not occurring.  See #30 

> This stops annoying situations where a fighter comes in and strafes the fuel trucks which makes the base unusable as those helicopters can now just spawn in and sling new ones from the CC.

I'm not sure if I'd define this as 'annoying', especially given how prolific FARPs are now.  Ideally the strategy would be to kill the trucks to prevent enemy helos spawning (with 0 fuel) and refuelling but keep the CC intact to take over.  This is the current tactic anyway given the unintended behviour of certain mechanics discussed above.

> Base neutralising that blocks the enemy from talking to the ground crew
As stated above, a lot of players are not aware of this, nor that it is inherent to DCS.  In addition, it seems as if the DDCS rules override this in terms of CC building (#30) and helipad truck spawning (#23), which is even more confusing as this will allow spawning following unpacking of a Base Logistics Crate, but not refueling.

Also confusing.  Ignoring airbases, How does helipad contesting for talking to ground crew work? BAse radius or helipad contact?

> Capture points and neutral airfields are captured by spawning CC's at them, as long as the CC is alive you should be able to spawn there. 

This makes capture points (FARPs) more powerful than neutral airbases in some respects, given that the area to inhibit slinging is larger for contesting than a smaller base with just helipads.

===

A lot of this really needs clarification/summarising.  It's a 40min wasted flight for players.  I try explain what I know and even as shown above, I'm unsure as well!

**Comment by afinegan** [comment]
I made it easy for the first run (and logic on how they work), main bases and bases that have defenses are called MOB, and bases with a single helipad, based on command center is a FOB(categorized on how the base/area actually works) - in the future, we can have more types, FACTORY, PRISON, SHIPYARD - this change will go out when I reset the map today or tomorrow and should make things a lot easier to figure out
